packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx

```diff
@@ -200,6 +200,13 @@ describe("GIVEN a Tooltip", () => {
     });
   });
 
+  describe("WHEN content is falsy", () => {
+    it("then tooltip should still display", () => {
+      cy.mount(<Open content={0} />);
+      cy.findByRole("tooltip").should("exist");
```

Comment:
We should check that 0 is displayed visually.

Also haven't checked but our tests need to cover:
- Falsy content
- Undefined content
- Empty string content
to make sure everything is covered